src/vm/vm_core.rs

```diff
@@ -441,7 +439,11 @@ impl VirtualMachine {
         self.opcode_assertions(&instruction, &operands)?;
 
         if let Some(ref mut trace) = &mut self.trace {
-            if let (RelocatableValue(ref pc), RelocatableValue(ref ap), RelocatableValue(ref fp)) = (
```

Comment:
Why did this change?

Reply:
This change was made to avoid having the import
```rust
...
use crate::relocatable::MaybeRelocatable;
use crate::relocatable::MaybeRelocatable::RelocatableValue;
...
```
Honestly it was just a personal aesthetic choice